systemd/openqa-gru.service

```diff
@@ -5,7 +5,7 @@ Wants=openqa-setup-db.service
 
 [Service]
 User=geekotest
-ExecStart=/usr/share/openqa/script/openqa gru -m production run
+ExecStart=/usr/share/openqa/script/openqa gru -m production run --reset-locks
```

Comment:
I suggest to make that the default. IMHO our systemd services should be as simple as possible so that we can also spawn a production environment more easily without systemd, e.g. when no systemd is available.

Reply:
Maybe later on when this has proven to be reliable. But for now i have to disagree, this is a big difference in behaviour from the normal Minion worker and it should be obvious.

Reply:
The main problem is that using this flag prevents the user from running more than one gru worker at a time. This is a huge difference to normal Minion workers, where you can spawn as many as you like on as many servers as you like.

Reply:
ok, fine